### DISCUSSION TITLE
Feature request: standalone installer

### DISCUSSION THREAD

**Discussion by EmberHeartshine** [File-New-Project/EarTrumpet#840] [Completed discussions]
I might be old-fashioned, but I've never been incredibly fond of centralized package management on Windows. The inability to install a given program offline is one of my pet peeves. I use Windows LTSC, which does not have the Windows Store (deliberately so), and instaling an entire package management framework for one program is just overkill.

I understand that chocolatey allows pre-downloading programs to install offline at a later time (internalizing), but what if I don't want that machine online at all? From what I understand of chocolatey, EarTrumpet's installer is just a zip file prepackaged with a Powershell script to install it. Would it be possible for that to be released standalone, for example through Github Releases?

**Comment by riverar** [comment]
Thanks for the submission.

This is a popular request; I recently outlined our rationale and covered installation options here https://github.com/File-New-Project/EarTrumpet/issues/1004#issuecomment-1010675687.

Locking thread.
